### PR TITLE
Update autoStart metatype name to Automatically start for clarity

### PR DESCRIPTION
**Pull request by alexsm82**
Update autoStart metatype name from Auto start to Automatically Start for clarity.

Issue #2035

### CODE REVIEW

**Comment by alexsm82** [comment]
#build

**Comment by alexsm82** [comment]
The build was red due to unrelated failures and infra issues, but all JCA and EJB MDB tests passed.  Ready for delivery once the ID review is complete.

File: dev/com.ibm.ws.jca.feature/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -42,7 +42,7 @@ authData=Authentication Data
 authData$Ref=Authentication data reference
 authData.desc=Default authentication data for an activation specification.
 
-autoStart=Auto start
+autoStart=Automatically start
```
Comment (chirp1) [inline dev/com.ibm.ws.jca.feature/resources/OSGI-INF/l10n/metatype.properties]: OK

File: dev/com.ibm.ws.jca/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -38,5 +38,5 @@ recoveryAuth=Recovery Authentication Data
 recoveryAuth$Ref=Recovery authentication data reference
 recoveryAuth.desc=Authentication data for transaction recovery.
 
-autoStart=Auto start
+autoStart=Automatically start
```
Comment (chirp1) [inline dev/com.ibm.ws.jca/resources/OSGI-INF/l10n/metatype.properties]: OK

**Pull request merged**